data/brands/amenity/bank.json

```
@@ -8078,6 +8078,18 @@
         "short_name": "PenFed"
       }
     },
+    {
+      "displayName": "Pennsylvania State Employees Credit Union",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "alt_name": "Pennsylvania State Employees Credit Union",
+        "amenity": "bank",
+        "brand": "PSECU",
+        "brand:wikidata": "Q7120626",
+        "name": "Pennsylvania State Employees Credit Union",
+        "short_name": "PSECU"
```

```suggestion
        "name": "PSECU"
```
`alt_name`/`name`/`short_name` should be different from one another. Based on their website it looks like they brand themselves as just 'PSECU' so I've suggested to change the name to this.